documentation/getting-started.md

```diff
@@ -4,7 +4,7 @@ title: Relax-and-Recover Quick Start
 ---
 
 This quick start guide will show you how to run Relax-and-Recover from the git
-checkout and create a bootable USB backup.
+checkout and create a bootable USB backup. Please note that this is intended for testing, experimentation and development as it runs ReaR in portable mode and therefore bypasses some of the [security measures](/documentation/security-architecture) that are in place for production use. For production use, please install ReaR into your system, either via a package or via `sudo make install` from source.
```

Comment:
I think ReaR is not run in portable mode here
because getting-started.md tells below
the user should do this (excerpts)
```
git clone https://github.com/rear/rear.git

cd rear/

sudo usr/sbin/rear format /dev/sdb

...

sudo usr/sbin/rear -v mkrescue
```

Reply:
Or leave out the portable mode part for now
because currently we do not check if
ReaR scripts are not owned by 'root'.

Reply:
Solved via
https://github.com/rear/rear.github.com/pull/20/commits/3d1db2307ff5e45414cbe718309b3704890d224f